test/suite/test_cursor18.py

```diff
@@ -455,10 +431,10 @@ def test_concurrent_insert(self):
         cursor2[1] = 2
         session2.commit_transaction("commit_timestamp=" + self.timestamp_str(5))
 
-        self.assertEqual(version_cursor.next(), 0)
-        self.assertEqual(version_cursor.get_key(), 1)
-        self.verify_value(version_cursor, 1, 1, 5, 5, 3, 0, 0)
-        self.assertEqual(version_cursor.next(), wiredtiger.WT_NOTFOUND)
+        self.assertEquals(version_cursor.next(), 0)
```

Comment:
The `assertEquals` change is the wrong way around -- that alias is removed in newer Python versions.